hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaControllerManagement.java

```diff
@@ -650,6 +654,8 @@ private void handleFinishedAndStoreInTargetStatus(final JpaAction action) {
             target.setUpdateStatus(TargetUpdateStatus.IN_SYNC);
         }
 
+        targetManagement.requestControllerAttributes(target.getControllerId());
+
```

Comment:
What about the `TargetUpdateStatus.IN_SYNC` from line 657. Do we need to fetch controller attributes even if the target is already in sync?